src/osx/carbon/cursor.cpp

```diff
@@ -278,7 +278,16 @@ wxCursor::wxCursor(const wxString& cursor_file, wxBitmapType flags, int hotSpotX
     if ( flags == wxBITMAP_TYPE_MACCURSOR_RESOURCE )
     {
 #if wxOSX_USE_COCOA
-        wxFAIL_MSG( wxT("Not implemented") );
+        wxString fileName = wxStandardPaths::Get().GetResourcesDir() + "/" + cursor_file + ".png";
+        wxImage image( fileName, wxBITMAP_TYPE_PNG );
+        if( image.IsOk() )
+        {
+            m_refData->DecRef() ;
+            m_refData = NULL ;
+            InitFromImage( image );
+        }
+        else
+            wxMessageBox( "No PNG cursor image found in Resources" );
```

Comment:
we should log an error IMHO, or wxFAIL_MSG, but definitely not pop-up a message box

Reply:
Yes, will change that. Probably use wxFAIL_MSG.
Do you think it will make sense to use CUR file as fall back?

Reply:
> Do you think it will make sense to use CUR file as fall back?

this would be a good idea especially as CUR does support multi-res - although my current macOS code does not yet translate this properly